proposals/description/fip_105.ts

```diff
@@ -1,17 +1,133 @@
 import { ProposalDescription } from '@custom-types/types';
 
 const fip_105: ProposalDescription = {
-  title: 'FIP-105: Grant DAI PSM skimmer PCV_CONTROLLER_ROLE',
+  title: 'FIP-105: Reinforce PCV by consolidating assets and perform technical maintenance',
   commands: [
+    /////////  DAI PSM Skimmer   //////////////
     {
       target: 'core',
       values: '0',
       method: 'grantRole(bytes32,address)',
       arguments: ['0x0866eae1216ed05a11636a648003f3f62921eb97ccb05acc30636f62958a8bd6', '{daiFixedPricePSMFeiSkimmer}'],
       description: 'Grant the new DAI PSM Skimmer the PCV_CONTROLLER_ROLE'
+    },
+
+    ////////    DPI LBP    ////////
+    {
+      target: 'dpi',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{dpiToDaiLBPSwapper}', '37888449801955370645659'],
+      description: 'Transfer DPI from DAO timelock to the LBP swapper'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{dpiToDaiLBPSwapper}', '187947000000000000000000'],
+      description: 'Withdraw DAI from the CompoundPCVDeposit and transfer to the LBP pool'
+    },
```

Comment:
what happens if DPI price changes? make sure to overfund DAI, or whitelist the swapper as a safe address

Reply:
Good point, have whitelisted as a safe address. Can then fine-tune as necessary